src/StreamingZipReader/StreamingZipEntry.cs

```diff
@@ -1,3 +1,3 @@
 namespace Wololo.StreamingZipReader;
 
-public readonly record struct StreamingZipEntry(string Name, uint Crc32, uint CompressedLength, uint Length);
\ No newline at end of file
+public readonly record struct StreamingZipEntry(string Name, uint Crc32, long CompressedLength, long Length);
```

Comment:
Off topic but since I can see it here, these days I'd make this a reference type (`sealed record` instead of `readonly record struct`). The main reason for me is so that you can't write `new StreamingZipEntry()` as valid code.

Reply:
Agreed and done.